identity-service/src/routes/healthCheck.js

```diff
@@ -4,11 +4,91 @@ const { handleResponse, successResponse, errorResponseServerError } = require('.
 const { sequelize } = require('../models')
 const { getRelayerFunds } = require('../txRelay')
 const Web3 = require('web3')
+
 const axios = require('axios')
 
 let notifDiscProv = config.get('notificationDiscoveryProvider')
 
+// Defaults used in relay health check endpoint
+const ONE_HOUR_AGO_BLOCKS = 720 // 5 blocks/sec = 720 blocks/hr
+const MAX_TRANSACTIONS = 100 // max transactions to look into
+const MAX_ERRORS = 5 // max acceptable errors for a 200 response
+const ACCOUNT = config.get('relayerPublicKey')
+
 module.exports = function (app) {
+  /**
+   * Relay health check endpoint. Takes the query params startBlock, endBlock, maxTransactions, and maxErrors.
+   * If those query params are not specified, use default values.
+   */
+  app.get('/health_check/relay', handleResponse(async (req, res) => {
+    const start = Date.now()
+    const audiusLibsInstance = req.app.get('audiusLibs')
+    const web3 = audiusLibsInstance.web3Manager.getWeb3()
+
+    let endBlockNumber = req.query.endBlock || (await web3.eth.getBlockNumber())
+    // In the case that no query params are defined and the endBlockNumber is less than 720, default startBlockNumber to 0
```

Comment:
this should never be a less than 720 realistically. i don't think we should need `defaultStartBlockNumber`